src/command_modules/azure-cli-consumption/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+0.2.0
+++++++++++
```

Comment:
Heading length is too long. It should match the no. of characters in the heading.
CI may fail due to this.